text/0006-dataset_naming.md

```diff
@@ -0,0 +1,97 @@
+- Feature Name: dataset_naming
+- Start Date: 
+- RFC PR: 
+- Issue: NA
+
+# Summary
+[summary]: #summary
+
+Define the qri naming system and conventions for name resolution.
+
+# Motivation
+[motivation]: #motivation
+
+Why are we doing this? What use cases does it support? What is the expected outcome?
```

Comment:
This is still the placeholder text.

How about:

"Qri uses a number of jargon terms around dataset names that need clarification and consistent usage."

Reply:
This RFC is supposedly putting fourth the concept of dataset naming in the first place. I'd propose:

> As a decentralized system, qri must confront the [Zooko's Triangle problem](https://en.wikipedia.org/wiki/Zooko%27s_triangle) which is establishing a way of referring to datasets that is:
> - human readable
> - decentralized
> - unique
> Because Qri is assumed to be built atop a content-address file system as it's storage layer, the properties of being decentralized & unique are already present.
> The Qri naming system maps a human readable name to the _tip_ of a dataset history.